frontend/src/Commons/Footer/Footer.style.ts

```diff
@@ -4,25 +4,27 @@
  */
 import styled from '@emotion/styled';
 import * as tokens from '@bcgov/design-tokens/js';
+import screenSizes from '@Constants/ScreenSize';
```

Comment:
ScreenSize is just an object and not a component or class. You should consider renaming the file to `screenSize`